### PR TITLE
fix: mf-4329 issues of wallet header

### PR DESCRIPTION
**Pull request by UncleBill**
https://mask.atlassian.net/browse/MF-4429
https://mask.atlassian.net/browse/MF-4430
https://mask.atlassian.net/browse/MF-4433
https://mask.atlassian.net/browse/MF-4434
https://mask.atlassian.net/browse/MF-4435
https://mask.atlassian.net/browse/MF-4436
https://mask.atlassian.net/browse/MF-4437
https://mask.atlassian.net/browse/MF-4438
https://mask.atlassian.net/browse/MF-4440
https://mask.atlassian.net/browse/MF-4443
https://mask.atlassian.net/browse/MF-4444
https://mask.atlassian.net/browse/MF-4445
https://mask.atlassian.net/browse/MF-4447
https://mask.atlassian.net/browse/MF-4454
https://mask.atlassian.net/browse/MF-4465
https://mask.atlassian.net/browse/MF-4470
https://mask.atlassian.net/browse/MF-4452

### CODE REVIEW

File: packages/mask/src/extension/popups/pages/Wallet/EditNetwork/index.tsx [hunk 1]
```diff
@@ -110,79 +109,122 @@ export const EditNetwork = memo(function EditNetwork() {
...
+                const issues = JSON.parse(message) as ZodCustomIssue[]
+                const isInvalid = issues.some((issue) => issue.path[0] !== 'currencySymbol')
+                if (!isInvalid) return true
+                issues.forEach((issue) => {
+                    // We assume there is no multiple paths.
+                    setError(issue.path[0] as keyof FormInputs, {
+                        message: t(issue.message as AvailableLocaleKeys, issue.params),
+                    })
+                })
+            } catch {}
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Wallet/EditNetwork/index.tsx:145]: `setError` in the catch clauses.

File: packages/mask/src/extension/popups/pages/Wallet/components/WalletHeader/UI.tsx [hunk 2]
```diff
@@ -33,15 +33,11 @@ const useStyles = makeStyles()((theme) => ({
...
-    avatar: {
-        marginRight: 4,
-        width: 30,
-        height: 30,
-    },
     nickname: {
         color: '#07101B',
         lineHeight: '18px',
         fontWeight: 700,
+        fontFamily: 'Helvetica',
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Wallet/components/WalletHeader/UI.tsx]: This font is only built-in on Mac. Try to use the global font setup.

File: packages/mask/src/extension/popups/pages/Wallet/EditNetwork/index.tsx (hunk 1 above)
Comment (UncleBill) [reply packages/mask/src/extension/popups/pages/Wallet/EditNetwork/index.tsx:145]: `setError` requires a field name.
This try-catch is to parse the json safely, it it fails we can ignore it.

File: packages/theme/src/Theme/dashboard/component-changes.tsx [hunk 3]
```diff
@@ -32,6 +32,9 @@ export const Grid: Theme = {
 
 const button = css_var('m-button', { main: 1, contrast: 1, light: 1 })
 export const Button: Theme = (mode, colors): ThemeOptions => ({
+    typography: {
+        fontFamily: 'Helvetica',
```
Comment (guanbinrui) [inline packages/theme/src/Theme/dashboard/component-changes.tsx:36]: I see some non-popup components used this font either. Set it here? 

`packages/theme/src/Theme/theme.ts`

Comment (guanbinrui) [inline packages/theme/src/Theme/dashboard/component-changes.tsx:36]: Please remove all other CSS rules are 'fontFamily: 'Helvetica' in the codebase.

Comment (UncleBill) [reply packages/theme/src/Theme/dashboard/component-changes.tsx:36]: OK. Removed that in popup

**Review by guanbinrui — APPROVED** [review]

File: packages/theme/src/Theme/dashboard/component-changes.tsx (hunk 3 above)
Comment (Jack-Works) [reply packages/theme/src/Theme/dashboard/component-changes.tsx:36]: there is no font fallback? how will Windows/Linux users see this

**Pull request merged**